packages/address/src/loader.ts

```diff
@@ -51,8 +54,8 @@ export async function loadCountry(
       query,
       operationName: GRAPHAL_OPERATION_NAMES.country,
       variables: {
-        countryCode,
-        locale,
+        countryCode: countryCode.toUpperCase(),
```

Comment:
This shouldn't be needed since all of the country codes in `SupportedCountry` are upper case already.